readme/api/get_started/plugins.md

```diff
@@ -7,41 +7,57 @@ In this article you will learn the basic steps to build and test a plugin in Jop
+This will setup source control.
+
+
+## Run Joplin in Development Mode
+
+You should test your plugin in Development Mode. Doing so means that Joplin will run using a different profile, so you can experiment with the plugin without risking to accidentally change or delete your data.
 
 ## Building the plugin
 
-The file `src/index.ts` already contain some basic code meant for testing the plugin. In particular it contains a call to [joplin.plugins.register](https://joplinapp.org/api/references/plugin_api/classes/joplinplugins.html), which all plugins should call to register the plugin. And an `onStart()` event handler, which will be executed by Joplin when the plugin starts.
+From the scaffolding, `src/index.ts` now contains the basic code for a Hello World plugin. 
+
+Two things to note:
+1. It contains a call to [joplin.plugins.register](https://joplinapp.org/api/references/plugin_api/classes/joplinplugins.html). All plugins call this to register the plugin in the app.
+2. An `onStart()` event handler method, which is called when the plugin starts.
 
 To try this basic plugin, compile the app by running the following from the root of the project:
 
 	npm run dist
 
-Doing so should compile all the files into the `dist/` directory. This is from here that Joplin will load the plugin.
-
-## Testing the plugin
-
-In order to test the plugin, you might want to run Joplin in [Development Mode](https://github.com/laurent22/joplin/blob/dev/readme/api/references/development_mode.md). Doing so means that Joplin will run using a different profile, so you can experiment with the plugin without risking to accidentally change or delete your data.
+Doing so should compile all the files into the `dist/` directory. This is where Joplin will load the plugin.
 
-Finally, in order to test the plugin, open the Setting screen, then navigate the the **Plugins** section, and add the plugin path in the **Development plugins** text field. For example, if your plugin project path is `/home/user/src/joplin-plugin`, add this in the text field.
+## Install the plugin
+Open Joplin `Settings/Preferences > Plugins` section. Under Advanced Settings, add the plugin path in the `Development plugins text field. 
+This should be the path to your main plugin directory, i.e. `path/to/your/root/plugin/directory`.
 
-Restart the app, and Joplin should load the plugin and execute its `onStart` handler. If all went well you should see the test message in the plugin console: "Test plugin started!".
+## Test the Plugin, Hello World!
+Restart the Development app from the command line/terminal, and Joplin should load the plugin and execute its `onStart` handler. If all went well you should see the test message in the plugin console: "Hello world. Test plugin started!". You will also be able to see the information from the manifest in the `Settings > Plugins`
```

Comment:
`Configuration > Plugins`

Reply:
By the way, should that be wrapped in a code block? By the way, could you make it bold rather than wrap it in a code block please? And same for above.

Reply:
On line 56 it still says Settings > Plugins as inline code.

Reply:
Whoops, fixed it.